blog/_posts/2021-08-20-symbiosis-symbulation.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Symbiosis in Symbulation"
+date: August 5, 2021
+author: Tiffany-Ellen Vo
+---
+
+# Symbiosis in Symbulation
+#### August 5, 2021 Tiffany-Ellen Vo
+- - - 
+
+[comment]: <> (How do I make it so the text is aligned with the picture?)
+
+><img src="https://scontent-sjc3-1.xx.fbcdn.net/v/t1.6435-9/106543962_3538908449476892_6632988677477612051_n.jpg?_nc_cat=110&ccb=1-4&_nc_sid=09cbfe&_nc_ohc=MVm9XlfPXKYAX8k78kl&_nc_ht=scontent-sjc3-1.xx&oh=aff9f72aa230c94ddce73a0e5ab28c3e&oe=6131225E" alt="drawing" width="200"/> Hello! My name is Tiffany-Ellen Vo, I'm currently a senior at UC Santa Cruz, studying computer engineering. I’m especially interested in logic design, robotics, and computer architecture. In my free time, I love rock climbing, baking, and crocheting. To see some of my other projects, check out my GitHub!
+
+- - - 
+
+## Introduction to Symbiosis 
+  Through the guidance of my mentor Anya, this summer I have worked on simulating symbiosis; symbiosis is any type of close and long-term biological interaction between two different biological organisims which would be mutualistic, commensalistic, or parasitic. We can look at the movie _Finding Nemo_ (2003) for an example of symbiosis; the mutualistic symbiotic relationship between an anemone and a clownfish is a classic one where both organisms benefit each other. The anemone provides the clownfish with shelter and protection, while the clownfish provides the anemone nutrients in the form of waste and scares off potential predator fish. Symbiotic relationships are important because they are a major driving force of evolution. However, parasitic and even mutualistic symbiosis relationships can make their species more vulnerable to environmental change if their partner starts having any problems. On the other hand, parasitism has been shown to lead 'arms races' where the two species are rapidly evolving to deal with each other.
+
+  While symbiosis has been simulated using Symbulation, the purpose of this project is to create and understand various types of complex symbiotic relationships while specificallly using the Avida complex genome to make the host and its symbionts.
+
+  For this summer, we focused on being able to add symbionts to a host system which would then increase the count of a point system. The original idea is that if the host reaches a certain amount of points, it would then donate a point to the symbiont and if the symbiont reaches a certain amount of points, then the symbiont would donate a point to the host which would then be multiplied by a synergy factor. The synergy factor represents the symbiont being able to provide resources to the host that it couldn't by itself, like the clownfish providing nutrients to the anemone that it wouldn't be able to get on its own.
```

Comment:
Undefined terms in first sentence: what is "host system" without symbionts and what is a "point system"? One sounds like an ecosystem(?), the other like a scoring system which seem very different but are referred to here with the same word.